packages/flutter/pubspec.yaml

```diff
@@ -19,5 +22,13 @@ dependencies:
   newton:
     path: ../newton
 
+dev_dependencies:
+  flx:
+    path: ../flx
```

Comment:
Looks like they're already there. Maybe we don't need this dependency? This was just me merging in the dependencies we had for the unit/ directory.